tests/jquery.flot.navigate.Test.js

```diff
@@ -278,6 +279,66 @@ describe("flot navigate plugin", function () {
         });
     });
 
+    describe('touchDrag', function() {
+      it('should drag the plot (start + move + end)',function() {
```

Comment:
Repeat this test for a different frameRate. You will need https://jasmine.github.io/2.0/introduction.html#section-Mocking_the_JavaScript_Timeout_Functions 

OR

Delete everything related to the framerate and add it back later.